generators/add-events/cloud-events/templates/publishEvents.test.js

```diff
@@ -0,0 +1,89 @@
+/*<% if (false) { %>
+Copyright 2019 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+<% } %>
+* <license header>
+*/
+
+jest.mock('@adobe/aio-sdk', () => ({
+  Core: {
+    Logger: jest.fn()
+  },
+  Events: {
+    init: jest.fn()
+  }
+}))
+
+const { Core, Events } = require('@adobe/aio-sdk')
+const mockEventsInstance = { publishEvent: jest.fn() }
+const mockLoggerInstance = { info: jest.fn(), debug: jest.fn(), error: jest.fn() }
+Core.Logger.mockReturnValue(mockLoggerInstance)
+
+Events.init.mockReturnValue(mockEventsInstance)
+
+const action = require('./<%= actionRelPath %>')
+
+beforeEach(() => {
+  Events.init.mockClear() // only clears calls stats
+  jest.clearAllMocks()
+  Core.Logger.mockClear()
+  mockLoggerInstance.info.mockReset()
+  mockLoggerInstance.debug.mockReset()
+  mockLoggerInstance.error.mockReset()
+})
+
+const fakeRequestParams = { organizationId: 'fakeOrgId', apiKey: 'fakeKey', providerId: 'fakeProvider', eventCode: 'fakeEventCode', payload: {hello: 'world'}, __ow_headers: { authorization: 'Bearer fakeToken' } }
```

Comment:
here organizationId will have to be moved to the __ow_headers according to the above comment